jx/build/JXMLCompiler.js

```diff
@@ -171,6 +172,57 @@ JXMLCompiler.prototype.extractScript = function(build_assets) {
 	}
 }
 
+JXMLCompiler.prototype.expandBraces = function(build_assets) {
+	var delta = {}, expanded;
+
+	visit(build_assets.template, function(path, node) {
+		var d;
+
+		for (var k in node) {
+			if (k == 'children') continue;
+
+			if (/^\s*\{\{.*\}\}\s*$/.test(node[k])) {
+				expanded = true;
+
+				if (!d) {
+					d = delta;
+
+					for (var i = 0; i < path.length; i++) {
+						d = d.children = d.children || {};
+						d = d[path[i]] = d[path[i]] || {};
+					}
+				}
+
+				var code = node[k];
+
+				d[k] = code;
+
+				delete node[k];
+			}
+		}
+	});
+
+	if (!expanded) return;
+
+	build_assets.braces = delta;
+
+	// TODO Parse JS properly
+	var code = JSON.stringify(delta).replace(/"\s*\{\{|\}\}\s*"/g, '');
+
+	build_assets.script += ';\n' +
+		'this.expandBraces = function(attr) { return ' + code + ' }';
```

Comment:
At runtime, the function does the abstract act of "expanding braces" so I think it's fine for now